### PR TITLE
Add moactl to OpenShift Origin CI

### PR DESCRIPTION
**Pull request by jharrington22**

### CODE REVIEW

File: ci-operator/config/openshift/moactl/OWNERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+approvers:
+- vkareh
+- jharrington22
+- jeremyeder
+options: {}
+approvers:
```
Comment (yeya24) [inline ci-operator/config/openshift/moactl/OWNERS:6]: Just curious, should this one be `reviewers`?

Comment (petr-muller) [reply ci-operator/config/openshift/moactl/OWNERS:6]: Yeah it should, or not be here at all :)

File: ci-operator/jobs/openshift/moactl/OWNERS [hunk 2]
```diff
@@ -0,0 +1,9 @@
+approvers:
```
Comment (RiRa12621) [inline ci-operator/jobs/openshift/moactl/OWNERS:1]: `approvers` is duplicated

File: ci-operator/config/openshift/moactl/OWNERS [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
```
Comment (RiRa12621) [inline ci-operator/config/openshift/moactl/OWNERS:1]: `approvers` is duplicated

**Comment by jeremyeder** [comment]
small nit from rick and this can be merged I supposed. https://issues.redhat.com/browse/OSD-4400

**Pull request merged**